### PR TITLE
[FSDE-18246] Migrate Expansion panel component to new Vue Documentation

### PR DESCRIPTION
**Pull request by lumdmtr**
https://ctl.atlassian.net/browse/FSDE-18246

### CODE REVIEW

File: src/documentation/.gitignore [hunk 1]
```diff
@@ -88,3 +88,6 @@ sw.*
 
 # Vim swap files
 *.swp
+
+# Local sensative data
+.npmrc
```
Comment (krzysztofdrozdowskilumen) [inline src/documentation/.gitignore:93]: Great idea! I think we should consider including that in other .gitignore files 👏

File: src/documentation/pages/components-wip/expansion-panel/examples/_bordered.vue [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+        | To render bordered panels, apply the property <code>bordered</code>.
+      pre.language-html
+        code(v-highlight="$data.codeSnippets.webcomponent" class="html")
+    </Wrapper>
+    <pre class="language-html" slot="code-vue">
+      <code v-highlight="$data.codeSnippets.vue" class="html"></code>
+    </pre>
+    <Wrapper slot="code-htmlblueprint">
+      <JSNeeded />
+      .p--text(class="chi-tab__description")
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components-wip/expansion-panel/examples/_bordered.vue:100]:
```suggestion
      .p--text.chi-tab__description
```

File: src/documentation/pages/components-wip/expansion-panel/examples/_no-step.vue [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+        | Step numbering can be omitted by removing the <code>step</code> property.
+      pre.language-html
+        code(v-highlight="$data.codeSnippets.webcomponent" class="html")
+    </Wrapper>
+    <pre class="language-html" slot="code-vue">
+      <code v-highlight="$data.codeSnippets.vue" class="html"></code>
+    </pre>
+    <Wrapper slot="code-htmlblueprint">
+      <JSNeeded />
+      .p--text(class="chi-tab__description")
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components-wip/expansion-panel/examples/_no-step.vue:103]: Same here

File: src/documentation/pages/components-wip/expansion-panel/examples/_state_icon.vue [hunk 4]
```diff
@@ -0,0 +1,321 @@
+<template lang="pug">
+  <ComponentExample title="State Icon" id="stateIcon" :tabs="exampleTabs" padding="-p--3 -p-lg--6" additionalClasses="-bg--grey-10">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/components-wip/expansion-panel/examples/_state_icon.vue:2]:
```suggestion
  <ComponentExample title="State Icon" id="state-icon" :tabs="exampleTabs" padding="-p--3 -p-lg--6" additionalClasses="-bg--grey-10">
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**